build/images/scripts/start_ovs

```diff
@@ -7,8 +7,57 @@ source /usr/share/openvswitch/scripts/ovs-lib
 CONTAINER_NAME="antrea-ovs"
 OVS_RUN_DIR="/var/run/openvswitch"
 OVS_DB_FILE="${OVS_RUN_DIR}/conf.db"
+OVS_LOGROTATE_CONF="/etc/logrotate.d/openvswitch-switch"
+
+hw_offload="false"
+log_file_max_num=0
+log_file_max_size=0
+
+function usage {
+    echo "start_ovs"
+    echo -e "  -h|--help\t\t \tPrint help message"
+    echo -e "  --hw-offload\t\t \tEnable OVS hardware offload"
+    echo -e "  --log_file_max_num=<uint> \tMaximum number of log files to be kept for an OVS daemon. Value 0 means keeping the current value"
+    echo -e "  --log_file_max_size=<uint> \tMaximum size (in megabytes) of an OVS log file. Value 0 means keeping the current value"
+}
 
-set -euo pipefail
+while (( "$#" )); do
+  case "$1" in
+    -h|--help)
+      usage
+      exit 0
+      ;;
+    --hw-offload)
+      hw_offload="true"
+      ;;
+    --log_file_max_num=*)
+      log_file_max_num=$1
+      log_file_max_num=${log_file_max_num#"--log_file_max_num="}
+      ;;
+    --log_file_max_size=*)
+      log_file_max_size=$1
+      log_file_max_size=${log_file_max_size#"--log_file_max_size="}
+      ;;
+    -*|--*) # unsupported flags
+      echo "Error: unsupported flag $1" >&2
+      exit 128
+      ;;
+    *) # standalone arguments are not supported
+      echo "Error: unsupported argument $1" >&2
+      exit 128
```

Comment:
curious about the choice of 128 as the exit code? aren't exit code > 127 typically used for specific purposes?

Reply:
Actually forgot to change it. Just changed to 1.